### PR TITLE
DXCDT-138: "Using as a CLI" doc page [2/10]

### PR DESCRIPTION
**Pull request by willvedd**
## ✏️ Changes

The second of a ten PR cascade to publish the content from the Deploy CLI's documentation overhaul. This one focuses on the "Using as a CLI" page.

## 🔗 References

- [Documentation content outline](https://oktawiki.atlassian.net/wiki/spaces/DXCDT/pages/2632975220/Deploy+CLI+Docs+Content+Outline)
- ["Using as a CLI" draft](https://oktawiki.atlassian.net/wiki/spaces/DXCDT/pages/2637893974/Using+as+a+CLI)

### CODE REVIEW

File: docs/using-as-cli.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Path. Specifies the target directory for configuration files to be written to.
+
+### `--config_file`, `-c`
+
+Path. Specifies the user-defined configuration file (`config.json`). Refer to the list of [all configurable properties](#).
+
+### `--format`, `-f`
+
+Options: yaml or directory. Determines the Refer to the list of all configurable properties.
```
Comment (sergiughf) [inline docs/using-as-cli.md:19]:
```suggestion
Options: **yaml** or **directory**. Determines the Refer to the list of all configurable properties.
```

Just a suggestion to make it stand out more.

File: docs/using-as-cli.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### `--export_ids`, `-e`
+
+Boolean: When enabled, will export the identifier fields for each resource. Default: `false`.
```
Comment (sergiughf) [inline docs/using-as-cli.md:23]:
```suggestion
Boolean. When enabled, will export the identifier fields for each resource. Default: `false`.
```

File: docs/using-as-cli.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Using as a CLI
+
+The Deploy CLI, can be used as a standalone command line utility. Doing so provides a simple way to manage your Auth0 tenant configuration in CI/CD workflows.
```
Comment (Widcket) [inline docs/using-as-cli.md:3]:
```suggestion
The Deploy CLI can be used as a standalone command line utility. Doing so provides a simple way to manage your Auth0 tenant configuration in CI/CD workflows.
```

File: docs/using-as-cli.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Path. Specifies the target directory for configuration files to be written to.
+
+### `--config_file`, `-c`
+
+Path. Specifies the user-defined configuration file (`config.json`). Refer to the list of [all configurable properties](#).
+
+### `--format`, `-f`
+
+Options: `yaml` or `directory`. Determines the Refer to the list of all configurable properties.
```
Comment (Widcket) [inline docs/using-as-cli.md:19]: > Determines the Refer to the list

This seems to be incomplete.

File: docs/using-as-cli.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
+# Using as a CLI
```
[the remark this answers is not in the document]
Comment (willvedd) [reply docs/using-as-cli.md:1]: Resolved because it's been added with #578

File: docs/using-as-cli.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Path. Specifies the user-defined configuration file (config.json). Refer to the list of [all configurable properties](#).
+
+### `--env`
+
+Boolean. Indicates if tools should ingest environment variables or not. Default: `true`.
+
+### `--proxy_url`, `-p`
+
+A url for proxying requests, only set this if you are behind a proxy.
```
Comment (Widcket) [inline docs/using-as-cli.md:68]:
```suggestion
A url for proxying requests. Only set this if you are behind a proxy.
```

File: docs/using-as-cli.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+Boolean. Indicates if tools should ingest environment variables or not. Default: `true`.
+
+### `--debug`
+
+Boolean. Enables more verbose error logging; useful during troubleshooting. Default: `false`.
+
+### `--proxy_url`, `-p`
+
+A url for proxying requests, only set this if you are behind a proxy.
```
Comment (Widcket) [inline docs/using-as-cli.md:35]:
```suggestion
A url for proxying requests. Only set this if you are behind a proxy.
```

File: docs/using-as-cli.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+Path. Specifies the location of the resource configuration files. For YAML formats, this will point to the `tenant.yaml` file, for directory formats, this will point to the resource configuration directory.
+
+### `--config_file`, `-c`
+
+Path. Specifies the user-defined configuration file (config.json). Refer to the list of [all configurable properties](#).
+
+### `--env`
+
+Boolean. Indicates if tools should ingest environment variables or not. Default: `true`.
```
Comment (Widcket) [inline docs/using-as-cli.md:64]:
```suggestion
Boolean. Indicates if the tool should ingest environment variables or not. Default: `true`.
```

File: docs/using-as-cli.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+Options: yaml or directory. Determines the file format of the exported resource configuration files. See: [Available Resource Config Formats](available-resource-config-formats).
+
+### `--export_ids`, `-e`
+
+Boolean. When enabled, will export the identifier fields for each resource. Default: `false`.
+
+### `--env`
+
+Boolean. Indicates if tools should ingest environment variables or not. Default: `true`.
```
Comment (Widcket) [inline docs/using-as-cli.md:27]:
```suggestion
Boolean. Indicates if the tool should ingest environment variables or not. Default: `true`.
```

**Pull request merged**